packages/graphql-language-service-types/src/index.js.flow

```diff
@@ -222,6 +222,16 @@ export type ContextToken = {
   style: string,
 };
 
+export type ContextTokenForCodeMirror = {
```

Comment:
looking at this more, `ContextToken` is only used by codemirror, and does not seem to be used by vscode/monaco at all, which is interesting.

This means that we don't need a different type for codemirror

This also explains why you need to consume the methods directly instead of instantiating the class

Reply:
https://github.com/graphql/graphiql/blob/de4005b0fd486d5d81b50dcdc226f990c181ec4b/packages/graphql-language-service-interface/src/GraphQLLanguageService.ts#L198

both here and with hover, we dont pass the 4th context token argument
this explains a lot!

Reply:
Thank you for your feedback! Should I change the code to make function `getTokenAtPosition` return a token which has the same type as `ContextToken` for codemirror? Because it currently returns a token which has a different type.
https://github.com/graphql/graphiql/blob/917c3931634cf27eb31a0383ff7c15ffd902d221/packages/graphql-language-service-interface/src/getAutocompleteSuggestions.ts#L391